docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Audit logs
+description: Learn how to track and analyze your account's activities by using the audit logs.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+
+> **info** Audit logs are available for [Enterprise plan](https://expo.dev/pricing) customers.
+
+Audit logs record actions made with Expo Application Services (EAS) by accounts. Recorded data includes information about the affected entities, the type of modification made to them, who performed the action, and when the activity occurred.
+
+Since audit logs can only be created and never modified or deleted, they serve as a source of truth to help monitor events and debug issues occurring within accounts.
```

Comment:
```suggestion
## Key points

- Audit logs can only be created and never modified or deleted, they serve as a source of truth to help monitor events and debug issues occurring within accounts.
```

All the info from this paragraph until the image, is isolated factual information. Let's convert this into a list.